helmfile.d/charts/gatekeeper/mutations/templates/tekton-pipelines.yaml

```diff
@@ -0,0 +1,219 @@
+apiVersion: mutations.gatekeeper.sh/v1
+kind: Assign
+metadata:
+  name: privilege-escalation-tekton-pipelines-restricted-init
```

Comment:
Can we just install a regular install of the podsecuritypolicy chart, tied to the same install condition as tekton?